### PR TITLE
Invite code bug

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

File: src/html/ui/community/settings.tpl.html [hunk 1]
```diff
@@ -140,7 +140,7 @@
       <div class="setting-item">
         <div class="half-column">
           <label>Invitation code</label>
-          <p>{{community.beta_access_code}}</p>
+          <p>{{community.beta_access_code || '&nbsp;'}}</p>
```
Comment (levity) [inline src/html/ui/community/settings.tpl.html]: does a plain old empty string work here too?

File: src/html/ui/community/invite.tpl.html [hunk 2]
```diff
@@ -81,7 +81,11 @@
           <button type="button" ng-click="edit()">Change</button>
```
Comment (levity) [inline src/html/ui/community/invite.tpl.html]: wait, we allow editing the code on this page? how did i miss that? that shouldn't be the case. specifically for the case in which all members are allowed to access this screen, they should not be able to also change the code. you can code in just that specific case if you want but also just dropping editing from this screen completely is fine with me too.

File: src/html/ui/community/settings.tpl.html (hunk 1 above)
Comment (robbiecarlton) [inline src/html/ui/community/settings.tpl.html]: No, the <p> isn't rendered with an empty string

File: src/html/ui/community/invite.tpl.html (hunk 2 above)
Comment (robbiecarlton) [inline src/html/ui/community/invite.tpl.html]: Fixed. Good to merge?

**Comment by robbiecarlton** [comment]
@levity :shipit:?

**Comment by levity** [comment]
:shipit:

**Pull request merged**